packages/orchestration/src/utils/start-helper.js

```diff
@@ -222,16 +224,21 @@ harden(provideOrchestration);
  */
 export const withOrchestration =
   (contractFn, opts) => async (zcf, privateArgs, baggage) => {
-    const { marshaller, ...allOrchPowers } = privateArgs;
+    const { marshaller: remoteMarshaller, ...allOrchPowers } = privateArgs;
     const { storageNode: _, ...requiredOrchPowers } = allOrchPowers;
     const { publishAccountInfo, chainInfoValueShape } = opts ?? {};
+
+    const cachingMarshaller = wrapRemoteMarshaller(remoteMarshaller);
```

Comment:
Do we need both this _and_ the change to the orchestration contracts?

Reply:
Yes, orchestration contracts currently get the original remote marshaller. That's what I want to change, but it requires typing heroics.